docs/nodes/logging.md

```diff
@@ -0,0 +1,174 @@
+---
+order: 7
+---
+
+## Logging
+
+Structured logging adds detail to logs and allows the node operator to better identify what they are looking for. Tendermint supports log levels on a global and per module basis. This allows the node operator to see only the information they need and the developer to hone in on specific changes they are working on. 
+
+## Configuring Log Levels
+
+
+There are three log levels, `info`, `debug` and `error`. These can be configured either through the command line via  `tendermint start --log-level ""` or within the `config.toml` file.
+
+```toml
+# Output level for logging, including package level options
+log-level = "info"
+```
+
+The default setting is a global `info` level. If you would like to set specific modules level log levels it can be done in the following format. 
```

Comment:
```suggestion
The default setting is a global `info` level. If you would like to set the log level for a specific module, it can be done in the following format: 
```